api4/cloud.go

```diff
@@ -174,3 +204,15 @@ func confirmCustomerPayment(c *Context, w http.ResponseWriter, r *http.Request)
 
 	ReturnStatusOK(w)
 }
+
+func setUserDataHeaders(c *Context, w http.ResponseWriter) error {
+	user, err := c.App.GetUser(c.App.Session().UserId)
+	if err != nil {
+		return err
+	}
+
+	w.Header().Add("MM_CLOUD_USER_ID", user.Id)
+	w.Header().Add("MM_CLOUD_USER_EMAIL", user.Email)
+
+	return nil
+}
```

Comment:
Shouldn't we need to set this in the request that is created when connecting to CWS at https://github.com/mattermost/enterprise/blob/f22d88ec7fcc3faea5258fba5282e600bba0cfc4/cloud/cloud.go#L229-L243

Reply:
Yes, I misunderstood the change. Thanks for the heads up!